### ISSUE TITLE
Standardize height prop

### ISSUE THREAD

**Issue by Matt-Butler** [cerner/terra-core#225]
We have several components that have the prop `height` but different implementations. This property should be more predictable across all components.

[string('tiny', 'small', 'medium', 'large', 'huge')](https://github.com/cerner/terra-core/blob/7f1d3732290e7335506c5a630e2ac616361cd9ff/packages/terra-progress-bar/src/ProgressBar.jsx#L13)
```
<ProgressBar height="tiny" value={15} />
```

[string](https://github.com/cerner/terra-core/blob/7f1d3732290e7335506c5a630e2ac616361cd9ff/packages/terra-icon/src/IconBase.jsx#L22)
```
<Icon height="50" width="50" />
```

[number](https://github.com/cerner/terra-core/blob/master/packages/terra-image/src/Image.jsx#L25)
```
<Image height="50" />
```

**Comment by Matt-Butler** [comment]
Move forward with height as html attribute usage
If we come to it, explore per component height props, e.g. iconHeight=”small”

**Comment by bjankord** [comment]
Fixed in #247
